torch/csrc/distributed/rpc/request_callback_no_python.cpp

```diff
@@ -629,7 +639,7 @@ void RequestCallbackNoPython::processRpc(
   }
 }
 
-Message RequestCallbackNoPython::handleError(
+IValue RequestCallbackNoPython::handleError(
```

Comment:
What's the reason for changing this return type? It doesn't seem to be strictly necessary...

Reply:
`handleError` was used in multiple places like this:

```
retFuture->markCompleted(handleError(e, request.type(), request.id()));
```

We can do the conversion in on the callsite, but doing it within `handleError` can help dedup some conversion code.